spec/snippets-spec.coffee

```diff
@@ -655,6 +651,43 @@ describe "Snippets extension", ->
           expect(editor.lineTextForBufferRow(7)).toBe "    }one t1 three"
           expect(editor.lineTextForBufferRow(12)).toBe "};one t1 three"
 
+    describe "when there are nested snippets", ->
+      it "tries to expand before moving to the next tabstop", ->
```

Comment:
This test fails. The keybindings are resolved in the wrong order, but only in tests, it works in real life. This is the reason that the next test fails too.